package.json

```diff
@@ -192,7 +192,7 @@
     ]
   },
   "dependencies": {
-    "@amityco/ts-sdk-react-native": "^6.17.0",
+    "@amityco/ts-sdk-react-native": "6.20.1-621b475.0",
```

Comment:
I would like to wait this version to be stable before merge. Or may be you can make a ew PR after a new release?